### PR TITLE
Fix Long Usernames on Team Card

### LINKED ISSUE #347: Teams with many users / long usernames run off page
Consider just putting each name on each line? (and then for long usernames....not sure what to do there....)

### PR DESCRIPTION
**Pull request by lowtorola**
Resolves #347
- Trimmed usernames to a maximum of 21 characters on `TeamCard` (arbitrary by what looked good)
- Added newline after each user
- Aligned users to left and added a lil padding for aesthetics

### CODE REVIEW

**Review by n8kim1 — DISMISSED** [review]

**Comment by n8kim1** [comment]
Looks good!
Your call abt this one, cuz it's high-effort: maybe, for padded names, include a "..." at the end? Perhaps that reduces user confusion

**Review by gaurav-arya — CHANGES_REQUESTED** [review]
Agree with @n8kim1 about `...`'s for spliced names

**Comment by gaurav-arya** [comment]
Could we also do something for the rankings page? See below:
image

And make sure to test this on mobile screens too (you can test this using the developer tools on your browser, e.g. for chrome: https://developer.chrome.com/docs/devtools/device-mode/)

**Review by gaurav-arya — APPROVED** [review]

**Review by gaurav-arya — CHANGES_REQUESTED** [review]
Actually, could we put the `trimUsername` function into a `utils/misc.js` file to avoid repeating it?

**Review by gaurav-arya — APPROVED** [review]

File: frontend/src/utils/misc.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export function trimUsername(username, length) {
```
Comment (gaurav-arya) [inline frontend/src/utils/misc.js:1]: For consistency, could we remove `export` here and add `export {trim username}` to the bottom of the file?

**Review by gaurav-arya — CHANGES_REQUESTED** [review]

**Review by gaurav-arya — APPROVED** [review]

**Pull request merged**